### DISCUSSION TITLE
RivaTuner Statistics Server (RTSS) shows the on-screen-display (OSD) twice and overlapped

### DISCUSSION THREAD

**Discussion by Taschenbuch** [blish-hud/Blish-HUD#919] [Troubleshooting]
---
id: rtss-osd-overlapping
title: RTSS shows overlapping OSD
description: Troubleshoot overlapping OSD and double FPS counters caused by RivaTuner Statistics Server (RTSS) injecting into Blish HUD.
---

## Issue
RivaTuner Statistics Server (RTSS) shows the on-screen-display (OSD), e.g. FPS, twice and overlapped. That happens because RTSS injects itself not only into Guild Wars 2 but as well into Blish HUD.

## Solution
In RTSS in the global profile set 'Application Detection Level' to 'None' . Add Guild Wars 2 exe as a new Profile and set 'Application Detection Level' to 'Low' for that profile.

**Comment by Shinokami007** [comment]
found out i couldn't make your steps to work for my case, so i fiddled around and hopefully i got the result i was striving for...
first off, i have a profile for blish and one for gw2.
on the blish profile i disabled the osd support, detection level on "none"
for gw2 i have the detection set to low and obviously the osd support is activated.
everything seems fine now.
